### DISCUSSION TITLE
Does webSocketTransport handle auto reconnect?

### DISCUSSION THREAD

**Discussion by JakubGrzegowski** [wevm/viem#673] [Question]
Gm

Was searching docs and issues for a while, but could not find any information if the webSocketTransport handles auto reconnects. Since websockets are known as fast but pretty unstable approaches it's important information. Anyone got idea?

**Comment by shlomich** [comment]
So I have done a test I am doing with every lib I am testing and noticed viem currently fails to reconnect. (I just disable my adapter and enable it to see what happens).
@jxom - Is there anything planned for that? I want to migrate our project and this is the only thing that is missing pretty much :)
